nipype/interfaces/fsl/model.py

```diff
@@ -1134,28 +1134,28 @@ class L2Model(BaseInterface):
 
     def _run_interface(self, runtime):
         cwd = os.getcwd()
-        mat_txt = ['/NumWaves       1',
-                   '/NumPoints      %d' % self.inputs.num_copes,
-                   '/PPheights      %e' % 1,
+        mat_txt = ['/NumWaves   1',
+                   '/NumPoints  {}'.format(self.inputs.num_copes),
+                   '/PPheights  1',
```

Comment:
Also I don't see anything about scientific notation (or anything about these specific lines) in the page you linked to, and I don't think it's the ideal reference for this code because it's describing how to write text files that will be converted to the actual files that FSL uses with an intermediate tool.